plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraLatestConnectorSmokeTest.java

```diff
@@ -36,14 +36,13 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     {
         switch (connectorBehavior) {
             case SUPPORTS_CREATE_SCHEMA:
+            case SUPPORTS_RENAME_TABLE:
+            case SUPPORTS_ROW_LEVEL_DELETE:
                 return false;
 
             case SUPPORTS_DELETE:
                 return true;
 
-            case SUPPORTS_ROW_LEVEL_DELETE:
-                return false;
```

Comment:
revert.

i intentionally resisted the temptation to group all false-s together

also, SUPPORTS_DELETE and SUPPORTS_ROW_LEVEL_DELETE are related, so they should be next to each other

Reply:
Why it's better not to group them? Arguably, grouping makes it easier to see what's supported and what's not (yet). Should there be a comment explaining this?

Reply:
Separate commit and PR. Then two of you can fight.

Reply:
I don't want to fight. I'll revert this. I just wanted to know the reason. It's not obvious to me.

Reply:
Reverted.